src/main/kotlin/io/gitlab/arturbosch/detekt/idea/config/DetektPluginSettings.kt

```diff
@@ -95,9 +94,9 @@ class DetektPluginSettings(
         var baselinePath by string()
         var pluginJars by list<String>()
 
-        //////////////////////////////////
-        //// Deprecated v2 properties ////
-        //////////////////////////////////
+        // ////////////////////////////////
+        // // Deprecated v2 properties ////
+        // ////////////////////////////////
```

Comment:
This seems an unintended change

Reply:
well yes and no; autocorrected by KtLin but I reverted them :)